doc/src/_static/custom.css

```diff
@@ -144,6 +144,26 @@ This can be removed once https://github.com/executablebooks/mdit-py-plugins/issu
     margin: 0 0.2em 0.25em -1.4em;
 }
 
+
+.sidebar-drawer {
+    transform: translateX(0);
+    transition: transform 0.3s ease-in-out;
+  }
+  
+.sidebar-drawer.collapsed {
+    transform: translateX(-100%);
+  }
+
+/* For screen sizes less than 857px the hamburger provided by Furro is used and
+for screen sizes greater than 857px our custom hamburger button is displayed */
+@media screen and (max-width: 857px) {
```

Comment:
![output circle-artifacts com_output_job_540094c2-0398-4369-8286-942ca1060539_artifacts_0_doc__build_html_index html](https://github.com/user-attachments/assets/4d5b016e-cf8c-4a05-a789-954ce714ac6f)

I'm not sure that this is correct breakpoint. I found issue with desktop with small browser width.